test/integration/suites/k8s-scratch/00-setup

```diff
@@ -40,7 +42,6 @@ log-info "starting cluster..."
 ./bin/kind create cluster --name k8stest --config ./conf/kind-config.yaml || fail-now "unable to create cluster"
 
 log-info "loading container images..."
-./bin/kind load docker-image --name k8stest spire-server-scratch:latest-local
+./bin/kind load docker-image --name k8stest spire-server:latest-local
```

Comment:
This integration test was meant to test the scratch images. Was there any particular reason you are changing this to the standard images?

Reply:
Oh good catch. This was a copy paste problem. Let me go over the changes to the 00-setup scripts with a fine toothed come to make sure I didn't botch something else. Thanks!